tests/test_bgp_af.rb

```diff
@@ -829,6 +833,6 @@ def test_table_map
     afs.each do |af|
       table_map(55, 'red', af)
     end
-    config('no route-policy sjc') if platform == :ios_xr
+    config_no_warn('no route-policy sjc') if platform == :ios_xr
```

Comment:
Good point. Took me a minute to see what's happening here. It's making noise because the route-policy can't be deleted while bgp still has a table-policy linked to it. 

The reason is due to this part of the test:

```
819     val = false
820     bgp_af.table_map_set('sjc', val)
821     refute(bgp_af.table_map_filter)
822
823    val = true
824    # TODO?  XR does not appear to support 'table-policy sjc filter'
825    if platform == :ios_xr

```

...the test wants to test for no filter, but the set has a side-effect of creating a `table-policy sjc` config. The next thing it does after that is bail out since ios_xr doesn't support filters. 

So I need to:
1) Add a validate_property check prior to that test
2) revert the config_no_warn() back to config()